serving/samples/helloworld-dart/README.md

```diff
@@ -0,0 +1,165 @@
+# Hello World - Dart sample
+
+A simple web app written in Dart that you can use for testing.
+It reads in the env variable `TARGET` and prints "Hello World: ${TARGET". If
+`TARGET` is not specified, it will use `"NOT SPECIFIED"` as `TARGET`.
```

Comment:
I just updated the other samples so that they display Hello World if not TARGET is specified, see https://github.com/knative/docs/pull/443. Can you also update yours?

Reply:
Oh, sorry I missed the feedback... should be done now :)